visidata/loaders/postgres.py

```diff
@@ -84,18 +88,61 @@ def postgresGetColumns(vd, cur):
         yield ColumnItem(coldesc.name, i, type=codeToType(coldesc.type_code, coldesc.name))
 
 
+class PgSchemataSheet(Sheet):
+    rowtype = 'schemas'
+
+    def reload(self):
```

Comment:
Generally we're standardizing on defining `iterload()`, which is almost entirely like `reload`, but you don't have to clear the rows, and instead of calling `addRow()` directly, just `yield` the row.